packages/jaeger-ui/src/components/TracePage/CriticalPath/utils/sanitizeOverFlowingChildren.test.js

```diff
@@ -35,11 +36,12 @@ function getExpectedSanitizedData(spans, test) {
 }
 
 describe.each([
-  [test3, new Map().set(test3.trace.spans[0].spanID, test3.trace.spans[0])],
-  [test4, new Map().set(test4.trace.spans[0].spanID, test4.trace.spans[0])],
+  [test3, new Map().set(test3.trace.spans[0].spanID, { ...test3.trace.spans[0], childSpanIds: [] })],
+  [test4, new Map().set(test4.trace.spans[0].spanID, { ...test4.trace.spans[0], childSpanIds: [] })],
   [test6, getExpectedSanitizedData(test6.trace.spans, 'test6')],
   [test7, getExpectedSanitizedData(test7.trace.spans, 'test7')],
   [test8, getExpectedSanitizedData(test8.trace.spans, 'test8')],
+  [test9, new Map().set(test9.trace.spans[0].spanID, { ...test9.trace.spans[0], childSpanIds: [] })],
```

Comment:
doesn't need to be part of this PR, but why L31-34 call getExpectedSanitizedData in the test setup? What it means is the test is effectively asserting `f(x) == f(x)`, which does not test anything, only increases coverage, but not in a meaningful way.

Reply:
I don't get your point, can you elaborate it. test6,test7,test8 are needed to test the cases like when child overflows from parent(like child end after parent or start before parent etc..)

Reply:
```
[test8, getExpectedSanitizedData(test8.trace.spans, 'test8')],
            ^^^^^^^^^^^^^^^^
```
I am referring to this part